ansible/roles/ocp4-workload-idm/tasks/remove_workload.yml

```diff
@@ -0,0 +1,62 @@
+---
+# Implement your Workload removal tasks here
+
+# Find IP of bastion
+- name: Gather VPC facts
+  ec2_vpc_net_facts:
+    filters:
+      tag:Name: "{{ aws_vpc_name }}"
+    region: "{{ aws_region }}"
+  register: vpc
+
+- name: Get instance facts
+  ec2_instance_facts:
+    filters:
+# FIXME - find a better way to discover the bastion name
+      "tag:Name": "{{ instances[0].name }}"
+    region: "{{ aws_region }}"
```

Comment:
please use `{{ aws_region_final | default(aws_region) }}`

Reply:
This location was refactored away, but I did change this for other locations of code.